File: blog-post-2/cpp/LeapYear.cpp
#include <iostream>
using namespace std;

int main()
{
    int year, isLeapYear;
    cout <<"Enter year: ";
    cin >> year;
    isLeapYear = 0;
    if (year % 4 == 0)
    {
        isLeapYear = 1;
        if (year % 100 == 0 && year % 400 != 0)
        {
            isLeapYear = 0;
        }
    }
    if (isLeapYear == 1)
    {
        cout <<"leap year";
    }
    else
    {
        cout <<"not leap year";
    }

    return 0;
}

File: blog-post-2/python/LeapYear.py
isLeapYear = False
year = int(input("Enter year: "))

if(year % 4 == 0):
    isLeapYear = True
    if year % 100 == 0 and year % 400 != 0:
        isLeapYear = False

if(isLeapYear):
    print("leap year")
else:
    print("not leap year")
